frontend/src/components/RecentOperations/OperationTypeColumn.tsx

```diff
@@ -0,0 +1,248 @@
+import React from "react";
+import BigNumber from "bignumber.js";
+
+import { FetchLastOperationsActionResponse } from "types";
+import { AssetLink } from "components/AssetLink";
+import { AccountBadge } from "components/AccountBadge";
+import { LiquidityPoolBadge } from "components/LiquidPoolBadge";
+
+interface OperationTypeColumnProps {
+  operation: FetchLastOperationsActionResponse;
+  horizonURL: string;
+}
+
+export const OperationTypeColumn: React.FC<OperationTypeColumnProps> = ({
+  operation,
+  horizonURL,
+}) => {
+  const amount = (
+    am: number,
+    asset_type: string,
+    asset_code: string,
+    asset_issuer: string,
```

Comment:
With so many params, it might be better to use an object here. We prefer to use camelCased names for params where possible (backend params could be the exception).

```suggestion
{
  amount,
  assetType,
  assetCode,
  assetIssuer,
}: FormatAmountProps
```